runtime/autoload/man.vim

```diff
@@ -118,15 +118,12 @@ endfunction
 " Expects a string like 'access' or 'access(2)'.
 function s:parse_page_and_section(sect, str) abort
   try
-    let save_isk = &iskeyword
-    setlocal iskeyword-=(,)
-    let page = substitute(a:str, '(*\(\k\+\).*', '\1', '')
-    let sect = substitute(a:str, '\(\k\+\)(\([^()]*\)).*', '\2', '')
+    let page = substitute(a:str, '\([-.[:alnum:]_]\+\).*', '\1', '')
+    let sect = substitute(a:str, '\([-.[:alnum:]_]\+\)(\([^()]*\)).*', '\2', '')
```

Comment:
No,

    let [page, sect] = matchlist(a:str, '\v\C([-.[:alnum:]_]+)\(([^()]*)\)')[1:2]

: using “very magic” mode reduces the number of backslashes and `\C` should simply be a convention, regardless of whether or not it changes something in the regular expression (`match*` functions depend on &ignorecase option, establishing such convention should prevent the possible damage of unexpected option value, just like `#` after comparison operators which do string comparison).